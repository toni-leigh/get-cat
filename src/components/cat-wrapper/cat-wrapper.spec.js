import React from 'react';
import { shallow } from 'enzyme';
import { createStore } from 'redux';
import CatWrapper from './cat-wrapper';

const store = createStore(() => {});

describe('<CatWrapper />', () => {
  let catWrapper;

  beforeAll(() => {
    catWrapper = shallow(
      <CatWrapper store={ store }/>
    ).dive();
  });

  test('basic render', () => {
    expect(catWrapper).toMatchSnapshot();
  });
});
