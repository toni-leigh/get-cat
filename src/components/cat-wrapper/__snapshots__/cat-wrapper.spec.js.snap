// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatWrapper /> basic render 1`] = `
<div
  className="crardx-cat-wrapper"
>
  Test
</div>
`;
